com.sap.cloud.lm.sl.mta/src/test/resources/com/sap/cloud/lm/sl/mta/resolvers/v3_1/system-parameters.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Can we extract the file on level /src/test/resources/com/sap/cloud/lm/sl/mta/resolvers and re-use it both for v2 and v3 resolvers?